language-examples/php/grpc/README.md

```diff
@@ -44,7 +44,7 @@ To run the example, do the following:
 
     `sh generate_proto_stubs.sh`
    
-   This will generate two folders containing Vectara proto php stubs: `Com/Vectara/` and `GPBMetadata/`
+   This will generate threee folders containing Vectara proto php stubs: `Com/Vectara/`, `GPBMetadata/`, and `Grpc/`
```

Comment:
Typo in `three`

Reply:
Addressed in 4dee05582ebc92ad5876afa58d2409b04fcd5252